src/decision/Criteria.jl

```diff
@@ -44,6 +44,15 @@ Base.@kwdef struct Criteria{P,N} <: EcoModel
         name="Outgoing Connectivity (Seed)",
         description="Higher values give preference to locations with high outgoing connectivity (i.e., provides larvae to other sites) for enhanced coral deployments.",
     )
+    seed_depth::P = Param(
```

Comment:
Just one last comment: in all other places you wrote `wt_depth_seed`, but here you wrote `seed_depth`. Do you think it would be ok to change this one to `depth_seed` as well, just to keep the pattern?

Reply:
I can do this. There's not consistency within Criteria with whether seed is at the end or beginning of the name but I can change it for this parameter

Reply:
It's never too late to start 😅